auditbeat/module/auditd/ecs_categorization.yml

```diff
@@ -0,0 +1,320 @@
+    # AUDIT_MAC_CONFIG_CHANGE - Changes to booleans
+    # AUDIT_MAC_UNLBL_ALLOW - NetLabel: allow unlabeled traffic
+    # AUDIT_MAC_CIPSOV4_ADD - NetLabel: add CIPSOv4 DOI entry
+    # AUDIT_MAC_CIPSOV4_DEL - NetLabel: del CIPSOv4 DOI entry
+    # AUDIT_MAC_MAP_ADD - NetLabel: add LSM domain mapping
+    # AUDIT_MAC_MAP_DEL - NetLabel: del LSM domain mapping
+    # AUDIT_MAC_IPSEC_EVENT - Audit an IPSec event
+    # AUDIT_MAC_UNLBL_STCADD - NetLabel: add a static label
+    # AUDIT_MAC_UNLBL_STCDEL - NetLabel: del a static label
+    # AUDIT_MAC_CALIPSO_ADD - NetLabel: add CALIPSO DOI entry
+    # AUDIT_MAC_CALIPSO_DEL - NetLabel: del CALIPSO DOI entry
+    # AUDIT_USER_ROLE_CHANGE - User changed to a new role // should this be classified EventTypeUserAccount?
+    # AUDIT_ROLE_ASSIGN - Admin assigned user to role // should this be classified EventTypeUserAccount?
+    # AUDIT_ROLE_REMOVE - Admin removed user from role // should this be classified EventTypeUserAccount?
+    # AUDIT_LABEL_OVERRIDE - Admin is overriding a label
+    # AUDIT_LABEL_LEVEL_CHANGE - Object's level was changed
+    # AUDIT_USER_LABELED_EXPORT - Object exported with label
+    # AUDIT_USER_UNLABELED_EXPORT - Object exported without label
+    # AUDIT_DEV_ALLOC - Device was allocated
+    # AUDIT_DEV_DEALLOC - Device was deallocated
+    # AUDIT_FS_RELABEL - Filesystem relabeled
+    # AUDIT_USER_MAC_POLICY_LOAD - Userspc daemon loaded policy
+    # AUDIT_ROLE_MODIFY - Admin modified a role
+    # AUDIT_USER_MAC_CONFIG_CHANGE - Change made to MAC policy
+    categories:
+    #   - policy
+
+  EventTypeAuditRule:
+    # AUDIT_SYSCALL - Syscall event
+    # AUDIT_PATH - Filename path information
+    # AUDIT_IPC - IPC record
+    # AUDIT_SOCKETCALL - sys_socketcall arguments
+    # AUDIT_SOCKADDR - sockaddr copied as syscall arg
+    # AUDIT_CWD - Current working directory
+    # AUDIT_EXECVE - execve arguments
+    # AUDIT_IPC_SET_PERM - IPC new permissions record type
+    # AUDIT_MQ_OPEN - POSIX MQ open record type
+    # AUDIT_MQ_SENDRECV- POSIX MQ send/receive record type
+    # AUDIT_MQ_NOTIFY - POSIX MQ notify record type
+    # AUDIT_MQ_GETSETATTR - POSIX MQ get/set attribute record type
+    # AUDIT_FD_PAIR - audit record for pipe/socketpair
+    # AUDIT_OBJ_PID - ptrace target
+    # AUDIT_BPRM_FCAPS - Information about fcaps increasing perms
+    # AUDIT_CAPSET - Record showing argument to sys_capset
+    # AUDIT_MMAP - Record showing descriptor and flags in mmap
+    # AUDIT_NETFILTER_PKT - Packets traversing netfilter chains
+    categories:
+    #   - ?
+
+  EventTypeCrypto:
+    categories:
+    #   - ?
+
+  EventTypeUnknown:
+    categories:
+    #   - ?
+
+messageTypes:
+  AUDIT_EXECVE:
+    categories:
+      - process
+    types:
+      - start
+  AUDIT_SERVICE_START:
+    categories:
+      - process
+    types:
+      - start
+  AUDIT_SERVICE_STOP:
+    categories:
+      - process
+    types:
+      - end
+  # are these mis-classified?
```

Comment:
> are these mis-classified?

I think the whole list of associations in this `messageTypes` structure looks good